packages/nimble-components/src/card/specs/README.md

```diff
@@ -0,0 +1,111 @@
+# Nimble Card
+
+## Overview
+
+The `nimble-card` is a container that is designed to contain arbitrary content specified by a client
+application. The `nimble-card` is intended for grouping related content.
+
+### Background
+
+[GitHub Issue #296: nimble-card component](https://github.com/ni/nimble/issues/296)
+
+An IxD and Visual Design spec for the `nimble-card` component have not yet been provided. We will add some
+preliminary styling in the interim.
+
+The new Routines UX in SystemLink Enterprise calls for grouping related configuration in a slide-out in labeled
+containers. See [Figma: NorthStar Vision for SLE Routines](https://www.figma.com/file/VQ7WIi3qqRG3r19VXqVvem/Stratus-Routines?type=design&node-id=1%3A74129&mode=design&t=KfPTwy4IhHbt42LR-1).
+We are designing the `nimble-card` component to address that use case.
+
+---
+
+## Design
+
+The `nimble-card` will not include any built-in content (title, footer, actions, etc.). Some component libraries
+do provide a card component with built-in, pre-styled content or pre-styled child components to use inside the card
+component. See the [Angular Material `mat-card` component](https://v5.material.angular.io/components/card/overview)
+for an example. We could add similar content in the future, but it is not required at this time.
+
+Since we're not providing any built-in content, the API surface area for the `nimble-card` is very small. The template
+contains only a slot, and there are no configurable properties on the component. Here is an example usage for
+the `nimble-card` component:
+
+```
+<nimble-card>
+    <div>
+        <h2>Heading</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        <nimble-button>Button</$nimble-button>
+    </div>
+</nimble-card>
+```
+
+While we don't have a finalized visual design spec, we expect the style will include a background color and a
+border, potentially with rounded corners. We will add design tokens for the background and border. The GitHub
+issue mentions several use cases for a card component that might call for different visual treatments, but we
+are only solving for the Routines use case at this time (grouping related configuration in a slide-out). To
+support other use cases in the future, we could potentially add appearance variants or make use of the `elevation`
```

Comment:
> or make use of the `elevation` property of the component.

I mentioned this in a different comment, but I didn't actually see an `elevation` property on the component. I'd take this last part of the sentence off and just leave it at "To
support other use cases in the future, we could potentially add appearance variants."